src/orb.yml

```diff
@@ -65,22 +65,45 @@ commands:
         description: Optional build command(s)
         type: steps
         default: []
+      yarn:
+        description: Use yarn instead of npm
+        type: boolean
+        default: false
     steps:
       - restore_cache:
           keys:
             - cache-{{ arch }}-{{ .Branch }}-{{ checksum "package.json" }}
       # it would be nice to vary this command based on presence of "package-lock.json"
       # and also allow using yarn or pnpm
-      - run:
-          name: 'Npm CI'
-          command: 'npm ci'
+      - when:
+          condition: << parameters.yarn >>
+          steps:
+            - run:
+                name: 'Yarn install'
+                command: 'yarn install'
+      - unless:
+          condition: << parameters.yarn >>
+          steps:
+            - run:
+                name: 'Npm CI'
+                command: 'npm ci'
       - run: npx cypress verify
       # save new cache folder if needed
-      - save_cache:
-          key: cache-{{ arch }}-{{ .Branch }}-{{ checksum "package.json" }}
-          paths:
-            - ~/.npm
-            - ~/.cache
+      - when:
+          condition: << parameters.yarn >>
+          steps:
```

Comment:
and especially that the steps could be `run` and `save_cache` and any other command